frontend/package.json

```diff
@@ -1,17 +1,34 @@
 {
   "name": "monitorfish-frontend",
+  "description": "Monitorfish web application.",
   "version": "0.1.0",
+  "license": "AGPL-3.0",
   "private": true,
+  "engines": {
+    "node": "18",
+    "npm": "8"
+  },
+  "scripts": {
+    "start": "sh -ac '. ../infra/configurations/frontend/.env.local; react-scripts start'",
+    "build": "sh -ac '. ../infra/configurations/frontend/.env.${REACT_APP_ENV}; react-scripts build'",
+    "build:dev": "REACT_APP_ENV=dev npm run build",
+    "build:prod": "REACT_APP_ENV=prod npm run build",
+    "build:local": "REACT_APP_ENV=local npm run build",
+    "lint": "eslint src/ --fix",
+    "type": "tsc",
+    "test": "jest --config=./config/jest.config.js --detectOpenHandles",
+    "cypress:open": "cypress open --browser firefox --e2e",
+    "cypress:open:electron": "cypress open --browser electron --e2e",
+    "cypress:run": "cypress run --browser firefox --e2e",
```

Comment:
Je n'arrive pas à lancer les tests en local : `Unknown file extension ".ts" for /Users/thomasbrosset/monitorfish/frontend/cypress.config.ts`

Reply:
Ca, pareil il faudra qu'on regarde ensemble ! Je l'ai testé sur un Mac et un PC Ubuntu sans soucis de mon côté.